AirPal/RootView.swift

```diff
@@ -26,12 +27,30 @@ struct RootView: View {
                 Text(viewModel.flightData?.arrival.iata ?? "Ari")
             }
             Text("Flight Status: \(viewModel.flightData?.flightStatus ?? "N/A")")
+            Button("Refresh") {
+                Task {
+                    do {
+                        flightChanged = try await viewModel.refreshFlightData()
+                    } catch {
+                        print("Error refreshing", error)
+                    }
+                }
+            }
+            if flightChanged {
```

Comment:
Does the View need to know if the flight is changed, in addition to if there are messages? Or: do we need both variables to achieve "when flight data changes, show message(s)"? 

It seems like there is extra state here:
|flightChanged|messages||
|---|---|---|
|false|[]|
|false|["a message"]| :warning: Shouldn't be possible |
|true|[]|:warning: Shouldn't be possible |
|true|["a message"]|

The two "not possible" cases tell me the view _could_ represent a state that shouldn't be possible. The easiest way to fix bugs is to prevent bugs! Rather than writing a test to ensure that doesn't happen, see if you can eliminate those cases from happening 👀